src/test/java/harness/TestContext.java

```diff
@@ -0,0 +1,54 @@
+package harness;
+
+import bio.terra.cli.context.GlobalContext;
+import bio.terra.cli.context.utils.Logger;
+import bio.terra.cli.service.ServerManager;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Comparator;
+
+/** Utility methods for manipulating the context during testing. */
+public class TestContext {
+
+  private TestContext() {}
+
+  /** Delete the global context directory. */
+  public static void deleteGlobalContext() throws IOException {
+    Path globalContextDir = GlobalContext.getGlobalContextDir();
+    if (!globalContextDir.toFile().exists()) {
+      return;
+    }
+
+    //   - walk the file tree and build a list of paths
+    //   - sort the list from longest paths to shortest
+    //   - delete the files in this order
+    // File.delete only works on file or empty directories
+    // since longer paths come first, all children of a directory will be deleted before we get to
+    // the directory
+    Files.walk(globalContextDir)
```

Comment:
I did find an implementation of this (recursive directory deletion) in the [Apache commons library,](https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/FileUtils.html#deleteDirectory(java.io.File)) but decided to just write the few lines of code here instead of including another library.